wgengine/magicsock/endpoint.go

```diff
@@ -907,25 +947,24 @@ func (de *endpoint) handlePongConnLocked(m *disco.Pong, di *discoInfo, src netip
 	}
 
 	if sp.purpose != pingHeartbeat {
-		de.c.dlogf("[v1] magicsock: disco: %v<-%v (%v, %v)  got pong tx=%x latency=%v pong.src=%v%v", de.c.discoShort, de.discoShort(), de.publicKey.ShortString(), src, m.TxID[:6], latency.Round(time.Millisecond), m.Src, logger.ArgWriter(func(bw *bufio.Writer) {
+		de.c.dlogf("[v1] magicsock: disco: %v<-%v (%v, %v)  got pong tx=%x latency=%v mtu=%v pong.src=%v%v", de.c.discoShort, de.discoShort(), de.publicKey.ShortString(), src, m.TxID[:6], latency.Round(time.Millisecond), pingSizeToWireMTU(sp.size, sp.to.Addr()), m.Src, logger.ArgWriter(func(bw *bufio.Writer) {
```

Comment:
Seems a bit weird to log the MTU computed from the ping size rather than the ping size itself. We know that a ping of that size got through, but we don't necessarily know that a larger packet would be dropped, right? I'd suggest to log the size itself.

Reply:
Yeah, that shouldn't be "mtu=" it should be something like "pkt len=". We do want to know the size of the packet that traversed the path - the ping size is just the ping message itself, and reading the log message we don't know whether it had IPv6 or IPv4 headers.